hilti/runtime/include/types/bitfield.h

```diff
@@ -13,18 +13,22 @@
 
 namespace hilti::rt {
 
+namespace trait {
+struct isBitfield {};
+} // namespace trait
+
 /// A bitfield is just a type wrapper around a tuple of the corresponding field
 /// values (including the hidden additional element storing the bitfield's
 /// original integer value). We wrap it so that we can customize the
 /// printing.
 template<typename... Ts>
-struct Bitfield {
+struct Bitfield : public trait::isBitfield {
     std::tuple<std::optional<Ts>...> value;
 };
 
 template<typename... Ts>
 Bitfield<Ts...> make_bitfield(Ts&&... args) {
-    return Bitfield<Ts...>{std::make_tuple(std::forward<Ts>(args)...)};
+    return Bitfield<Ts...>{{}, std::make_tuple(std::forward<Ts>(args)...)};
```

Comment:
Just a note, no action needed.

Another way do deal with the base class would be to use designated initializers,
```suggestion
    return Bitfield<Ts...>{.value = std::make_tuple(std::forward<Ts>(args)...)};
```